src/components/farmModal.html

```diff
@@ -325,28 +325,9 @@ <h5 class="p-0 modal-title text-muted" data-bind="text: `#${(Number($data) + 1).
                                         <tbody>
                                             <tr>
                                                 <td class="p-0">
```

Comment:
`class="p-0"` was added here to disable padding for the berrydex preview. It's not working since you've changed the list to a template.

Reply:
It is fixed now. Thank you for checking.